docs/configuration/tips.mdx

```diff
@@ -8,12 +8,18 @@ sidebar_position: 4
 
 ## Roles
 
-It is strongly recommended to keep your [ROLE](/docs/configuration/radio/device#roles) set to `CLIENT`. Only change this if you have a specific, well-understood reason to do so.
-
-### Why `CLIENT` is Recommended
-
-- `CLIENT` nodes efficiently repeat and route packets as needed.
-- They use smart delays for rebroadcasting, improving network stability.
+It is strongly recommended to keep your [ROLE](/docs/configuration/radio/device#roles) set to `CLIENT` or `CLIENT_MUTE`. Only use other roles if you have a specific, well-understood reason to do so. Read our [blog post about choosing the best role](/blog/choosing-the-right-device-role/).
+
+### Recommended roles: `CLIENT` and `CLIENT_MUTE`
```

Comment:
Sorry to be a stickler here but we really do try to keep the headings short for the TOC. Can you please adjust this to just be `Recommended Roles`?